lib/gc/Analysis/TargetDescriptionAnalysis.cpp

```diff
@@ -0,0 +1,87 @@
+//===-- TargetDescriptionAnalysis.cpp - target description impl -*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "gc/Analysis/TargetDescriptionAnalysis.h"
+#include <limits>
+#include <llvm/Support/Debug.h>
+
+namespace mlir {
+namespace gc {
+
+#define DEBUG_TYPE "target-description-analysis"
+
+// default values for properties
+static llvm::DenseMap<StringRef, int64_t> CPUTargetDeafultValueMap = {
+    {CPUTargetDescriptionAnalysis::kNumThreads, 1},
+    {CPUTargetDescriptionAnalysis::kL1CacheSize, 32 * 1024},
+    {CPUTargetDescriptionAnalysis::kL2CacheSize, 32 * 32 * 1024},
```

Comment:
Change it to `1024 * 1024` for better readability.

Reply:
Thanks for the advice! Changed it to 1024 base